keanu-python/keanu/vertex.py

```diff
@@ -30,9 +79,9 @@ def __parse_args(args):
     @staticmethod
     def __parse_arg(arg):
         if isinstance(arg, np.ndarray):
-            return Const(arg).unwrap()
+            return kn.Const(arg).unwrap()
         elif isinstance(arg, numbers.Number):
-            return arg
+            return Vertex.__parse_arg(np.array([[arg]]))
```

Comment:
If we make all numbers passed to a vertex ndarray, `__infer_tensor_from_scalar` in tenso.py is never reached

Reply:
Oh, nevermind it can still be used in `observe`

Reply:
I think it was my mistake - but could you fix the import statement for `observe`?

Reply:
done